sdk/storage/azure-storage-files-datalake/src/datalake_utilities.cpp

```diff
@@ -1,12 +1,12 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
-#include "azure/storage/files/datalake/internal/datalake_utilities.hpp"
+#include "private/datalake_utilities.hpp"
```

Comment:
nit: why is this one here, rather than together with the other private header below?

Reply:
Storage sdk is following headers order and grouping rule in Google C++ Style Guide.
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes